db/migrate/20150827194622_add_enable_reset_to_variant_overrides.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnableResetToVariantOverrides < ActiveRecord::Migration
+  def change
+    add_column :variant_overrides, :enable_reset, :boolean
```

Comment:
Just thinking about the naming of this boolean. Seeing as it represents a state, I think a name like "resettable", "reset_enabled" or even "can_be_reset" are perhaps better names? These allow us to use rails magic more effectively to do stuff like vo.resettable?, vo.can_be_reset? to be more explicit about the type of information we are asking for. For example, I think vo.resettable? is perhaps more readable than vo.enable_reset, or even vo.enable_reset?, both of which look like function calls to me.

Thoughts?